packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -9,53 +9,56 @@ meta:
 
 ## Reference user attributes
 
-When you create an Okta expression, you can reference any attribute that lives on an Okta User Profile or Application User Profile.
+When you create an Okta expression, you can reference any attribute that lives on an Okta user profile or application user profile.
 
-### Okta User Profile
+### Okta user profile
 
-Every user has an Okta User Profile. The Okta User Profile is the central source of truth for the core attributes of a User. To reference an Okta User Profile attribute, specify `user.` and the attribute variable name. For a list of core User Profile attributes, see [Default Profile properties](/docs/reference/api/users/#default-profile-properties).
+Every user has an Okta user profile. The Okta user profile is the central source of truth for the core attributes of a user. To reference an Okta user profile attribute, specify `user.` and the attribute variable name. For a list of core user profile attributes, see [Default profile properties](/docs/reference/api/users/#default-profile-properties).
 
 | Syntax            | Definitions                                                                   | examples                                                       |
 | --------          | ----------                                                                    | ------------                                                   |
-| `user.$attribute` | `user` reference to the Okta User<br>`$attribute` the attribute variable name | user.firstName<br>user.lastName<br>user.login<br>user.email |
+| `user.$attribute` | `user` reference to the Okta user<br>`$attribute` the attribute variable name | user.firstName<br>user.lastName<br>user.login<br>user.email |
 
-> **Note:** You can also access the User ID for each user with the following expression: `user.getInternalProperty("id")`.
+> **Note:** You can also access the user ID for each user with the following expression: `user.getInternalProperty("id")`.
 
-### Application User Profile
+### Application user profile
 
-In addition to an Okta User Profile, all Users have a separate Application User Profile for each of their applications. Application User Profiles store application-specific information about Users, such as the application `userName` or user `role`. To reference an Application User Profile attribute, specify the application variable and the attribute variable in the user profile of the application. In specifying the application, you can either name the specific application you're referencing or use an implicit reference to an in-context application.
+In addition to an Okta user profile, all users have a separate application user profile for each of their applications. Application user profiles store application-specific information about users, such as the application `userName` or user `role`.
```

Comment:
```suggestion
In addition to an Okta user profile, all users have a separate application user profile for each of their apps. Application user profiles store app-specific information about users, such as the app `userName` or user `role`.
```

Reply:
Leaving this instance of "application" because it sounds like an "application user profile" is a specific thing?

Reply:
I checked the docs and "app user" and "app user profile" seem to be regularly used. I'll change it!